skimage/util/_remap.pyx

```diff
@@ -17,6 +21,7 @@ def _map_array(np_anyint[:] inarr, np_numeric[:] outarr,
     # The prange option gave some compilation warnings
     #  "Unsigned index type not allowed before OpenMP 3.0"
     # and didn't seem to be any faster
-    # for i in prange(n_array, nogil=True): #
-    for i in range(n_array):
-        outarr[i] = lut[inarr[i]]
+    for i in prange(n_array, nogil=True): #
```

Comment:
maybe. I do not remember Cython rules for this now.

Reply:
Seems likely, since we are not touching any Python objects other than the arrays. On the other hand, since this function will be using all cores, making it nogil might result in thread contention and worse performance? 🤔